gssapi/raw/cython_converters.pyx

```diff
@@ -29,6 +29,13 @@ cdef gss_OID_set c_get_mech_oid_set(object mechs):
 cdef object c_create_oid_set(gss_OID_set mech_set, bint free=True):
     """Convert a GSS OID set struct to a set of OIDs"""
 
+    if mech_set == GSS_C_NO_OID_SET:
+        # return the empty set if the we get passed the C equivalent
+        # (it could be argued that the C equivalent is closer to None,
+        # but returning None would make the API harder to work with,
+        # without much value)
+        return set()
+
```

Comment:
If we ever need the distinction we may need to make an incompatible API change and start returning None, is that ok ?

Reply:
yes, but we'll have to rev the major version.  I'm curious as to what that situation might be -- you're not using this to detect errors (we have errors for that), so fundamentally, what's the difference here between "a set with no items", and "no set" from the user perspective.

Reply:
ok